### PR TITLE
Updated Chinese version of README

### PR DESCRIPTION
**Pull request by simanw**

### CODE REVIEW

**Comment by frank-zsy** [comment]
Sorry, we do not support `puml` files with Chinese character, you may leave the `puml` file with English version, and the Chinese version README should be `README.md` and the English version should be `README_en.md`, can you modify it?

**Comment by simanw** [comment]
Sure

File: README_en.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# data-sync
+
+![travis](https://travis-ci.com/wuhan2020/data-sync.svg?branch=master)
+
+武汉新型冠状病毒防疫信息收集平台-数据同步服务
```
Comment (frank-zsy) [inline README_en.md]: Is this line also need translation?

**Comment by frank-zsy** [comment]
Thanks

**Pull request merged**